backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java

```diff
@@ -521,7 +521,7 @@ private void writeCpuFlags(String[] typeAndFlags) {
 
     private void writeCpuTune(boolean numaEnabled) {
         writer.writeStartElement("cputune");
-        Map<String, Object> cpuPinning = vmInfoBuildUtils.parseCpuPinning(VmCpuCountHelper.isAutoPinning(vm) ?
+        Map<String, Object> cpuPinning = vmInfoBuildUtils.parseCpuPinning(VmCpuCountHelper.isDynamicCpuPinning(vm) ?
```

Comment:
how about extracting this change to a separate PR? this can be quickly merged and remove so much noise from the PR..